src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMockFactory.java

```diff
@@ -40,20 +40,13 @@ private String getRandomName()
 	 */
 	private String getUniqueRandomName()
 	{
-		if (usedNames.size() >= 100)
+		String name;
+		do
 		{
-			throw new RuntimeException("Out of names");
-		}
-
-		while (true)
-		{
-			String name = getRandomName();
-			if (!usedNames.contains(name))
-			{
-				usedNames.add(name);
-				return name;
-			}
+			name = getRandomName();
 		}
+		while (usedNames.contains(name));
```

Comment:
It should add this new name to already used names here
```suggestion
		while (usedNames.contains(name));
		usedNames.add(name);
```